contributors/devel/godep.md

```diff
@@ -15,6 +15,20 @@ the tools.
 
 This doc will focus on predictability and reproducibility.
 
+## Justifications for an update
+
+Before you update a dependency, take a moment to consider why it should be 
+updated. Valid reasons include:
+ 1. We need new functionality that is in a later version.
+ 2. New APIs in the dependency are better than the old ones.
+ 3. Bugs were fixed that impact Kubernetes.
+ 4. Security issues were fixed even if they don't impact Kubernetes yet.
+ 5. Performance, scale, or efficiency was meaningfully improved.
+ 6. Testing improvements.
```

Comment:
I think you are right below - this one is dubious.

Reply:
> I think you are right below - this one is dubious.

Removed.  Bug fixes are already covered, so if the tests found bugs, we'd already get it.